specs/latest/2.0/index.html

```diff
@@ -2123,6 +2123,9 @@ <h2 class="no-toc">Table of contents</h2>
         </table>
         <p>If <em>pname</em> is not in the table above, generates an <code>INVALID_ENUM</code> error.</p>
         <p>If <em>sampler</em> is not a valid sampler object, generates an <code>INVALID_OPERATION</code> error.</p>
+        <p>Modifying a parameter of a sampler object affects all texture units to which that sampler object is bound.
+        Calling <code>TexParameter</code> has no effect on the sampler object bound to the active texture unit.
+        It will modify the parameters of the texture object bound to that unit.</p>
```

Comment:
Personally, it's still a little confused from this sentence. As I see, there're two parts of parameters for a texture unit. One is sampling related like `wrap_mode` and `filter`, another is not. While a sampler is bound to a texture unit, the texture unit will ignore its settings of sampling part and use equivalent parameters of this sampler. If my understanding is correct, then it's better not to use `texParameter` after binding a sampler unless modifying the parameters except sampling parts.
So I suggest to change this sentence to `It will modify the parameters except those in the table above of the texture object bound to that unit.` What do you think about this?